tests/manager/searchCustomer/managerCanSearchCustomerByLastName.spec.js

```diff
@@ -30,6 +43,14 @@ Test:
 3. Assert customer row is present in the table. 
 4. Assert no other rows is present in the table.
 */
+const bankManagerMainPage = new BankManagerMainPage(page); 
+const openAccountPage = new OpenAccountPage(page); 
+const customersListPage= new CustomersListPage (page);
+
+await bankManagerMainPage.open();
+await bankManagerMainPage.clickCustomersButton();
+await customersListPage.waitForOpened();
+await customersListPage.searchCustomerByLastName(lastName);
```

Comment:
The test case is missing assertions to verify that the customer row is present and no other rows are present in the table after searching by last name. Consider adding assertions to check these conditions.